homework-submission/todo-app/src/component/week3Components/gender.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const Gender = ({ gender }) => <li>{gender}</li>;
```

Comment:
Add a cool icon! Maybe just use an emoji?

Reply:
I have fixed it in week4 homework.